contrib/debian/changelog

```diff
@@ -1,6 +1,12 @@
+zcash (1.0.7+1) jessie; urgency=medium
```

Comment:
1.0.7+1 should arguably mean "The first Debian-specific repackaging of upstream 1.0.7." Although we can't use 1.0.7-1 as the Debian version number (because of the restriction that "[If there is no debian_revision then hyphens are not allowed](https://www.debian.org/doc/debian-policy/ch-controlfields.html)"), we could use 1.0.7-1+0.